website/docs/docs/cloud/manage-access/set-up-snowflake-oauth.md

```diff
@@ -95,6 +95,10 @@ When clicking on the `Connect Snowflake Account` successfully redirects you to t
 * Your user might not have access to the Snowflake role defined on the development credentials in dbt Cloud. Double-check that you have access to that role and if the role name has been correctly entered in as Snowflake is case sensitive.
 * You're trying to use a role that is in the [BLOCKED_ROLES_LIST](https://docs.snowflake.com/en/user-guide/oauth-partner.html#blocking-specific-roles-from-using-the-integration), such as `ACCOUNTADMIN`.
 
+#### The requested scope is invalid
+When clicking on the `Connect Snowflake Account` successfully redirects you to the Snowflake login page, but you receive an `The requested scope is invalid` error. This could mean:
```

Comment:
```suggestion
When you select the `Connect Snowflake Account` button to try to connect to your Snowflake account, you might get an error that says `The requested scope is invalid` even though you were redirected to the Snowflake login page successfully. 

This error might be because of a configuration issue in the Snowflake OAuth flow, where the `role` in the profile config is mandatory for each user. This means each user needs to supply their role information, regardless of whether it's provided in the project connection page.
```